ot/bregman.py

```diff
@@ -1723,6 +1724,11 @@ def empirical_sinkhorn(X_s, X_t, reg, a=None, b=None, metric='sqeuclidean',
         Max number of iterations
     stopThr : float, optional
         Stop threshol on error (>0)
+    isLazy: boolean, optional
+        If True, then only calculate the cost matrix by block and return the dual vectors
+        If False, calculate full cost matrix and return outputs of sinkhorn function.
+    batchSize: int or tuple of 2 int, optional
+        Shape of the block of cost matrix, when isLazy=True
```

Comment:
Size of the batcheses used to compute the sinkhorn update without memory overhead. When a tuple is provided it sets the size of  the left/right  batches